SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/CultureInfoHelper.cs

```diff
@@ -36,11 +37,18 @@ public static CultureInfo GetCurrentCulture()
 
         private static string InvokeGetLocaleInfoEx(string lpLocaleName, uint LCType)
         {
-            var buffer = new StringBuilder(BUFFER_SIZE);
-            var resultCode = GetLocaleInfoEx(lpLocaleName, LCType, buffer, BUFFER_SIZE);
-            if (resultCode > 0)
+            try
+            {
+                var buffer = new StringBuilder(BUFFER_SIZE);
+                var resultCode = GetLocaleInfoEx(lpLocaleName, LCType, buffer, BUFFER_SIZE);
+                if (resultCode > 0)
+                {
+                    return buffer.ToString();
+                }
+            }
+            catch (DllNotFoundException exception)
             {
-                return buffer.ToString();
+                AppCenterLog.Verbose(AppCenterLog.LogTag, $"Failed to call GetLocaleInfoEx: {exception.Message}");
```

Comment:
I guess the log level should be `Debug` or `Warning`. @guperrot Which one do you prefer?

Reply:
It should be a warning yes.

Reply:
After talking to Ivan offline, we can leave it as debug if that always happens when bound to Unity.